ietf-layer0-types.yang

```diff
@@ -1508,7 +1675,7 @@ module ietf-layer0-types {
         "OIF-400ZR-01.0: Implementation Agreement 400ZR";
     }
     leaf tx-polarization-power-difference {
-      type power-in-db;
+      type power-in-dbm;
```

Comment:
Power difference is a ratio, implying in dB. I see in #135 that _decimal-2-digits-or-null_ seems the chosen solution.

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/70/commits/0038f5e771db23c9e2109f149b30d3226340db67

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>